cmd/serve/headers_test.go

```
@@ -0,0 +1,123 @@
+	} {
+		if got := h.Get(name); got != "" {
+			t.Errorf("expected %s to be absent when both metadata args are nil; got %q", name, got)
+		}
+	}
+}
+
+func TestSetBAMLHeaders_PlannedOnly(t *testing.T) {
+	t.Parallel()
+
+	h := http.Header{}
+	max := 3
```

golangci-lint (2.11.4)

[error] variable max has same name as predeclared identifier

(predeclared)